src/components/NodeLink/NodeLink.vue

```diff
@@ -23,9 +23,13 @@ export default {
       type: Object,
       default: () => {}
     },
-    nodeLabel: {
+    labelVariable: {
       type: String,
-      default: "id"
+      default: "_key"
+    },
+    colorVariable: {
+      type: String,
+      default: "_key"
```

Comment:
This should be

```suggestion
      default: "table"
```

right?